app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    no_mask = 0
+    problems = False
+    for human in friends:
+        try:
+            cafe.visit_cafe(human)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            problems = True
+            no_mask += 1
+
+    if no_mask:
+        return f"Friends should buy {no_mask} masks"
+
+    if not problems:
+        return f"Friends can go to {cafe.name}"
```

Comment:
you can make it without "problems". use "no_mask" > 1 to return f"Friends should buy {no_mask} masks" and else: return f"Friends can go to {cafe.name}"
remove problems variable